loopchain/protos/loopchain.proto

```diff
@@ -18,6 +18,11 @@ service PeerService {
     rpc AddTxList (TxSendList) returns (CommonReply) {}
     // broadcast vote on proposed block
     rpc VoteUnconfirmedBlock (BlockVote) returns (CommonReply) {}
+
+    // exchange height info
+    rpc BlockHeightRequest (HeightRequest) returns (CommonReply) {}
+    rpc BlockHeightResponse (HeightResponse) returns (CommonReply) {}
+    rpc BlockInformationRequest (HeightResponse) returns (CommonReply) {}
```

Comment:
I think.... BlockRequest is better.

Reply:
Changed:
- BlockInformationRequest -> BlockRequest